packages/venia-concept/src/components/SearchBar/__tests__/autocomplete.spec.js

```diff
@@ -8,7 +8,7 @@ import Autocomplete from '../autocomplete';
 jest.mock('src/classify');
 jest.mock('@magento/peregrine');
 jest.mock('../suggestions', () => () => null);
-jest.doMock('react-apollo/ApolloContext', () => React.createContext());
+jest.doMock('react-apollo', () => React.createContext());
```

Comment:
Since the import is now 

```js
import { ApolloContext } from 'react-apollo';
```

doesn't this mock have to return an object with an `ApolloContext` property?

```js
jest.doMock('react-apollo', () => ({ ApolloContext: React.createContext() }));
```

Reply:
Actually, that mock isn't even need anymore. Will remove.